homeassistant/components/philips_js/__init__.py

```diff
@@ -30,6 +30,8 @@
     Platform.LIGHT,
     Platform.REMOTE,
     Platform.SWITCH,
+    Platform.BINARY_SENSOR,
+    Platform.SENSOR,
```

Comment:
PRs should be a small as possible.

This PR should be split into three separate PRs:

- One bumping the library version
- One adding the binary sensor platform
- One adding the sensor platform

Reply:
Library was bumped already in https://github.com/home-assistant/core/pull/94811
I changed this PR and the other one to support binary_sensor only.

A new one for sensor is following.